indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -402,8 +402,15 @@
       this.newButtonItem = document.createElement('la-gutter-item');
       this.newButtonItem.appendChild(this.newButton);
 
-      this.gutter = this.el.querySelector('.gutter la-gutter');
-      this.gutter.akomaNtoso = this.el.querySelector('.content-with-gutter > .content');
+      this.gutter = this.el.querySelector('la-gutter');
+
+      /*
+      * la-gutter components should have the akoma-ntoso as a attribute set. In the case that it is not dont. Find
+      * the the nearest akomo-ntoso element
+      * */
+      if(!this.gutter.akomaNtoso) {
```

Comment:
I'm trying to work out when this would happen - surely it would be a bug? If a gutter isn't linked to its la-akoma-ntoso, that's a problem.

Reply:
ooking in la-web-components: it already defaults to the first la-akoma-ntoso on the page: https://github.com/laws-africa/la-web-components/blob/master/src/utils/linking.ts#L11

so we could drop this completely?